geoportal/vars.yaml

```diff
@@ -139,6 +139,7 @@ vars:
         #   - https://localhost:3001/webcomponents/index.ts
 
         sitnFeedbackPath: custom/feedback
+        pytreeLidarprofileJsonUrl: https://sitn.ne.ch/pytree
```

Comment:
Also, should be removed from desktop_alt

Reply:
So in default and not in both ?

Reply:
desktop_alt extends desktop, and we don't need it on mobile, then only here :-)

Reply:
Done.